test/example.test.js

```diff
@@ -3,16 +3,62 @@
 
 const test = QUnit.test;
 
-test('time to test a function', (expect) => {
+test('test doesUserWin(player, computer) input and outputs align with game rules R>S, P>R, S>P, like=like(draw)', (expect) => {
     //Arrange
     // Set up your arguments and expectations
-    const expected = true;
+    //Is it ok to use arrays in testing?
```

Comment:
I'm sure you noticed today, but you can use arrays for testing.  :) Realized you probably fixed this a bit with your pair programming, but you'll get more practice with arrays in the future.